src/Trakx.Circle.ApiClient/CircleApiConfiguration.cs

```diff
@@ -1,13 +1,18 @@
-﻿using Trakx.Utils.Attributes;
+﻿using System.ComponentModel.DataAnnotations;
+using Trakx.Utils.Attributes;
 
 namespace Trakx.Circle.ApiClient
 {
     public record CircleApiConfiguration
     {
 #nullable disable
+        [Required]
         public string BaseUrl { get; init; }
-        [SecretEnvironmentVariable]
+        [AwsParameter,SecretEnvironmentVariable]
```

Comment:
👍 so once you have that, you can get configuration from AWS
BUT at the moment the variable has not been defined
I can show you how to proceed on a call